src/main/java/net/atos/zac/signalering/event/SignaleringEventObserver.java

```diff
@@ -62,17 +62,26 @@ public class SignaleringEventObserver extends AbstractEventObserver<SignaleringE
     @Override
     public void onFire(final @ObservesAsync SignaleringEvent<?> event) {
         try {
-            LOG.fine(() -> String.format("Signalering event ontvangen: %s", event.toString()));
+            LOG.fine(() -> String.format("Signalering event ontvangen: %s", event));
             event.delay();
+
             final Signalering signalering = buildSignalering(event);
-            if (signalering != null && signaleringenService.isNecessary(signalering, event.getActor())) {
-                final SignaleringInstellingen subscriptions = signaleringenService.readInstellingen(signalering);
-                if (subscriptions.isDashboard()) {
-                    signaleringenService.createSignalering(signalering);
-                }
-                if (subscriptions.isMail()) {
-                    signaleringenService.sendSignalering(signalering);
-                }
+            if (signalering == null) {
+                LOG.fine(() -> String.format("Cannot build signalering from event ontvangen: %s", event));
```

Comment:
When we can't parse the event, is that not cause for a warning? Or do we expect this to happen regularly?